### PR TITLE
removed -f flag for remote

### PR DESCRIPTION
**Pull request by devidask27**
sudo ./pf9ctl check-node --help
Check if a node satisfies prerequisites to be ready to be added to a Kubernetes cluster. Read more
	at https://platform9.com/blog/support/managed-container-cloud-requirements-checklist/

Usage:
  pf9ctl check-node [flags]

Flags:
  -h, --help              help for check-node
  -i, --ip strings        IP address of host to be prepared
  -p, --password string   ssh password for the nodes
  -s, --ssh-key string    ssh key file for connecting to the nodes
  -u, --user string       ssh username for the nodes

Global Flags:
      --verbose   print verbose logs

### CODE REVIEW

**Comment by devidask27** [comment]
TC build link
http://teamcity.platform9.horse:8111/buildConfiguration/Pf9project_Platform9Components_Pf9ctl/1440824

File: cmd/checkNode.go [hunk 1]
```diff
@@ -22,8 +22,8 @@ func init() {
 	checkNodeCmd.Flags().StringVarP(&user, "user", "u", "", "ssh username for the nodes")
 	checkNodeCmd.Flags().StringVarP(&password, "password", "p", "", "ssh password for the nodes")
 	checkNodeCmd.Flags().StringVarP(&sshKey, "ssh-key", "s", "", "ssh key file for connecting to the nodes")
-	checkNodeCmd.Flags().StringSliceVarP(&ips, "ips", "i", []string{}, "ips of host to be prepared")
-	checkNodeCmd.Flags().BoolVarP(&floatingIP, "floating-ip", "f", false, "")
+	checkNodeCmd.Flags().StringSliceVarP(&ips, "IPs", "i", []string{}, "IPs of host to be prepared")
```
Comment (anupbarve) [inline cmd/checkNode.go:25]: Since we are supporting a single IP, suggest to make it IP instead of IPs.

File: cmd/checkNode.go [hunk 2]
```diff
@@ -22,8 +22,8 @@ func init() {
...
+	//checkNodeCmd.Flags().BoolVarP(&floatingIP, "floating-ip", "f", false, "")
```
Comment (anupbarve) [inline cmd/checkNode.go:26]: Please put a comment here as to why we are commenting it out. This is unsupported in first version.

**Review by anupbarve — COMMENTED** [review]
Please put the console output in the description field.

File: cmd/prepNode.go [hunk 3]
```diff
@@ -34,8 +34,8 @@ func init() {
 	prepNodeCmd.Flags().StringVarP(&user, "user", "u", "", "ssh username for the nodes")
 	prepNodeCmd.Flags().StringVarP(&password, "password", "p", "", "ssh password for the nodes")
 	prepNodeCmd.Flags().StringVarP(&sshKey, "ssh-key", "s", "", "ssh key file for connecting to the nodes")
-	prepNodeCmd.Flags().StringSliceVarP(&ips, "ips", "i", []string{}, "ips of host to be prepared")
-	prepNodeCmd.Flags().BoolVarP(&floatingIP, "floating-ip", "f", false, "")
+	prepNodeCmd.Flags().StringSliceVarP(&ips, "IPs", "i", []string{}, "IPs of host to be prepared")
+	//prepNodeCmd.Flags().BoolVarP(&floatingIP, "floating-ip", "f", false, "")
```
Comment (sahil-lakhwani) [inline cmd/prepNode.go:38]: same comment as Anup's for the check-node

**Comment by devidask27** [comment]
sudo ./pf9ctl check-node --help
Check if a node satisfies prerequisites to be ready to be added to a Kubernetes cluster. Read more
	at https://platform9.com/blog/support/managed-container-cloud-requirements-checklist/

Usage:
  pf9ctl check-node [flags]

Flags:
  -h, --help              help for check-node
  -i, --ip strings        IP address of host to be prepared
  -p, --password string   ssh password for the nodes
  -s, --ssh-key string    ssh key file for connecting to the nodes
  -u, --user string       ssh username for the nodes

Global Flags:
      --verbose   print verbose logs

**Comment by devidask27** [comment]
sudo ./pf9ctl check-node --help
Check if a node satisfies prerequisites to be ready to be added to a Kubernetes cluster. Read more
	at https://platform9.com/blog/support/managed-container-cloud-requirements-checklist/

Usage:
  pf9ctl check-node [flags]

Flags:
  -h, --help                                 help for check-node
  -i, --ip strings                          IP address of host to be prepared
  -p, --password string             ssh password for the nodes
  -s, --ssh-key string                ssh key file for connecting to the nodes
  -u, --user string                      ssh username for the nodes

Global Flags:
      --verbose   print verbose logs

**Review by anupbarve — APPROVED** [review]

**Pull request merged**